install fails on void linux

mdns fails to install on void linux with `fatal error: dns_sd.h: No such file or directory`

```
> mdns@2.4.0 install /home/yann/node_modules/mdns
> node-gyp rebuild

make : on entre dans le répertoire « /home/yann/node_modules/mdns/build »
  CXX(target) Release/obj.target/dns_sd_bindings/src/dns_sd.o
In file included from ../src/dns_sd.cpp:1:
../src/mdns.hpp:32:10: fatal error: dns_sd.h: No such file or directory
 #include <dns_sd.h>
          ^~~~~~~~~~
compilation terminated.
make: *** [dns_sd_bindings.target.mk:162: Release/obj.target/dns_sd_bindings/src/dns_sd.o] Error 1
make : on quitte le répertoire « /home/yann/node_modules/mdns/build »
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:193:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:255:12)
gyp ERR! System Linux 4.19.40_1
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/yann/node_modules/mdns
gyp ERR! node -v v11.14.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm WARN enoent ENOENT: no such file or directory, open '/home/yann/package.json'
npm WARN yann No description
npm WARN yann No repository field.
npm WARN yann No README data
npm WARN yann No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mdns@2.4.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mdns@2.4.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/binding.gyp
+++ b/binding.gyp
@@ -23,7 +23,8 @@
                  ]
     , 'conditions': [
         [ 'OS!="mac" and OS!="win"', {
-            'libraries': [ '-ldns_sd' ]
+            'include_dirs': [ '/usr/include/avahi-compat-libdns_sd/' ] # dns_sd.h is installed to /usr/include/avahi-compat-libdns_sd/ in Void Linux
+          , 'libraries': [ '-ldns_sd' ]
         }]
       , [ 'OS=="mac"', {
             'defines': [ 'HAVE_DNSSERVICEGETADDRINFO' ]
@@ -34,8 +35,8 @@
         }]
       , ['OS=="win"', {
             'variables': {
-                'BONJOUR_SDK_DIR': '$(BONJOUR_SDK_HOME)', # Preventing path resolution problems by saving the env var in variable first 
-                'PLATFORM': '$(Platform)' # Set  the platform
+                'BONJOUR_SDK_DIR': '$(BONJOUR_SDK_HOME)', # Preventing path resolution problems by saving the env var in variable first
+                'PLATFORM': '$(Platform)' # Set the platform
               }
           , 'include_dirs': [ '<(BONJOUR_SDK_DIR)/Include' ]
           , 'defines': [ 'HAVE_DNSSERVICEGETADDRINFO' ]

